### ISSUE TITLE
Geyser Extensions listed in plugin history and performance screen marker

### ISSUE THREAD

**Issue by DrHuckinJohn** [plan-player-analytics/Plan#3422]
### I would like to be able to..

If possible, I would like any installed extensions in Geyser (or any other plugins that support dedicated extensions) to also show as a subset of the Geyser (or other) plugin. Like is shown with plugins in the performance information screen, I would also like to see the plugin extension shown with a timestamp for update/install. Thanks!

### Is your feature request related to a problem? Please describe.

I installed the MCXboxBroadcast extension for Geyser today and have had a server behind tick warning during operations related to that extension. Would be nice to be able to easily & automatically data track that within the performance view of the Plan webserver.

**Comment by AuroraLS3** [comment]
How and where are these installed?

**Comment by DrHuckinJohn** [comment]
> How and where are these installed?
How: a .jar is dropped into the extensions folder like a standard paper plugin. It then installs on next server restart.
Where: On my paper 1.20.4 server: PaperMCserverFolder/plugins/Geyser-Spigot/extensions/

**Comment by AuroraLS3** [comment]
Is there some Floodgate or Geyser plugin also installed? (So that I could have some API to check the installed plugins)?

**Comment by DrHuckinJohn** [comment]
> Is there some Floodgate or Geyser plugin also installed? (So that I could have some API to check the installed plugins)?

Yes, I have Geyser installed and the extension needs it to run. The extension I am using specifically is located here: https://github.com/rtm516/MCXboxBroadcast
